sdk/servicebus/service-bus/src/log.ts

```diff
@@ -82,6 +85,14 @@ export interface ServiceBusLogger extends AzureLogger {
   logError(err: Error | AmqpError | undefined, ...args: any[]): void;
 }
 
+function isRetryableError(error: any): error is AmqpError {
+  if (!rheaIsAmqpError(error) && !isError(error)) {
+    return false;
+  }
+  const translatedError = translateServiceBusError(error);
+  return isMessagingError(translatedError) && translatedError.retryable;
```

Comment:
Could this be simplified to
```suggestion
  return translatedError?.retryable;
```

Reply:
I remember I needed to add the isMessagingError assertion so that `retryable` is available. The translate call returns `ServiceBusError | Error`